### PR TITLE
Adds missing operators for Variables and int scalars

### LINKED ISSUE #1202: __rmul__ (etc.) with int promotes to float
```python
import scipp as sc
var = sc.Variable(dims=['x'], values=[1,2,3])
var * 1 # ok, int
1 * var # not ok, float
```
Same for `__radd__` etc.

### PR DESCRIPTION
**Pull request by DanNixon**
Fixes #1202

### CODE REVIEW

File: python/tests/test_variable.py [hunk 1]
```diff
@@ -1010,3 +1010,27 @@ def test_equal():
 def test_not_equal():
     var = sc.Variable()
     assert_export(sc.not_equal, x=var, y=var)
+
+
+def test_radd_int():
+    var = sc.Variable(dims=['x'], values=[1,2,3])
+    assert (var + 1).dtype == sc.dtype.int64
```
Comment (SimonHeybrock) [inline python/tests/test_variable.py:1017]: I remember Python ints are different on different platforms (32 bit on Windows?), but maybe that was an older Python? See also comment below.

File: python/variable.cpp [hunk 2]
```diff
@@ -184,17 +184,31 @@ of variances.)");
       .def(
           "__radd__", [](Variable &a, double &b) { return a + b * units::one; },
           py::is_operator())
+      .def(
+          "__radd__", [](Variable &a, int &b) { return a + b * units::one; },
+          py::is_operator())
```
Comment (SimonHeybrock) [inline python/variable.cpp:189]: Should be `int64_t` to match the Python test? But may be correct actually, if it matches the Python `int` behavior.

File: python/tests/test_variable.py [hunk 3]
```diff
@@ -1010,3 +1010,27 @@ def test_equal():
 def test_not_equal():
     var = sc.Variable()
     assert_export(sc.not_equal, x=var, y=var)
+
+
+def test_radd_int():
+    var = sc.Variable(dims=['x'], values=[1, 2, 3])
+    assert (var + 1).dtype == sc.dtype.int64
+    assert (1 + var).dtype == sc.dtype.int64
```
Comment (SimonHeybrock) [inline python/tests/test_variable.py:1018]: So on Windows `var` will be `int32` I believe, and the addition promotes to `int64`? I think we may have this issue in other, existing overloads as well, so not necessarily something that needs to be fixed here.

Comment (DanNixon) [reply python/tests/test_variable.py:1018]: Beyond duplicating the exports (for both `int32` and `int64`) I'm not sure of an alternative.

Comment (SimonHeybrock) [reply python/tests/test_variable.py:1018]: I suppose that is what we might need to do, but I suggest we can leave that for another time.

**Comment by SimonHeybrock** [comment]
@DanNixon Can you fix the builds here?

**Pull request merged**